tutum/http.go

```diff
@@ -11,7 +11,8 @@ import (
 var customUserAgent = "go-tutum/" + version
 
 func SetUserAgent(name string) string {
-	customUserAgent = name + " " + customUserAgent
+	customUserAgent = ""
```

Comment:
It was what causes the user agent to repeat the name. As it's a global variable, if weave-daemon restarts, it will rewrite again this variable by adding the user agent.